src/controllers/dlgprefcontroller.cpp

```diff
@@ -249,6 +229,15 @@ void DlgPrefController::slotDirty() {
     m_bDirty = true;
 }
 
+const QString DlgPrefController::presetDisplayName(const PresetInfo& info) {
+    if (QFileInfo(info.getPath()).absoluteDir() == userPresetsPath(m_pConfig)) {
+        //: Shown in combobox to select controller mappings to distinguish user-installed mappings from mappings included in Mixxx
+        return info.getName() + " (" + tr("user") + ")";
```

Comment:
User is really not self explaining.
How about (user folder)

Reply:
Or just an asterisk and explain it below.

Reply:
:/ I don't think either of these are very good options. I am thinking we should revert to the 2.0 behavior, which sorted mappings by file path. This put mappings in the user folder at the top in alphabetical order, followed by the system mappings in alphabetical order. I do not recall any complaints about that in 2.0. That behavior was broken by effbb061a0b0f55c5ece9e5c1df2f5665cca7817 because it removed the QMap of mappings indexed by a QString for the file path.